.yarnrc.yml

```diff
@@ -1,11 +1,17 @@
 nodeLinker: node-modules
 
+enableGlobalCache: false
```

Comment:
Some might wonder, why it's suddently needed. Cause afterall it's overridden in the install action.

Context:

- By removing compression... local archives won't be zipped. That's what we want on CI... on local it's okay (more disk space, let's say it's fine). But if enableGlobalCache is true, downloaded archives won't be pruned... and over time  the loca install will blow. See my answer in https://github.com/actions/setup-node/issues/325#issuecomment-1535094446.

PS: enableGlobalCache will be default in upcoming version, so we align for the future as well)